src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -943,24 +943,13 @@ public bool Save([@NotNull]MutableDocument document, [@NotNull]Func<MutableDocum
         /// <param name="blob">The blob object will be saved into Database.</param>
         public void SaveBlob(Blob blob)
```

Comment:
I really want to see warnings all over this to not use it for general cases ( @pasin )

Reply:
Let's do in the the API spec so others could see it too. At the very least, the API doc should say something.